### DISCUSSION TITLE
Getting "Exception Processing Message 0xc0000005 - Unexpected parameters" error after changing from Direct3d11 to OpenGL

### DISCUSSION THREAD

**Discussion by He3dshot** [ppy/osu#28135] [Q&A] [answered]
I changed the Graphics API from Direct3D11 to OpenGL and now osu!lazer won't launch. I play on an integrated gpu. I tried reinstalling the game multiple times but it still wouldn't work. I even went ahead to look for registry keys in the registry but even deleting the ones i found didn't help.

![awdawd](https://github.com/ppy/osu/assets/76886441/61b8e6b0-3fb3-40e1-852f-42fe5d6b0868)

**Comment by Joehuu** [comment, marked as answer]
Does deleting `framework.ini` in `%AppData%/osu` work?

**Reply by He3dshot** [reply]
Yes!
